tiledb/sm/rest/rest_client.cc

```diff
@@ -1661,6 +1664,20 @@ RestClient::post_consolidation_plan_from_rest(
       serialization_type_, returned_data);
 }
 
+std::unordered_map<std::string, std::string> RestClient::load_headers(
+    const Config& cfg) {
+  std::unordered_map<std::string, std::string> ret;
+  auto iter = ConfigIter(cfg, constants::rest_header_prefix);
```

Comment:
Can't this be in the first part of the for loop?

Reply:
What do you mean?

Reply:
This line goes away, second line becomes:
for (auto iter = ConfigIter(cfg, constants::rest_header_prefix); !iter.end(); iter.next()) {

Reply:
Done.